pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -782,11 +794,13 @@ func (ku *Upgrader) copyCustomNodeProperties(client armhelpers.KubernetesClient,
 		for k, v := range oldNode.Annotations {
 			if _, ok := newNode.Annotations[k]; !ok {
```

Comment:
this has diverged from Aks upgrade in two ways. 
1) AKS doesn't copy 
2) AKS always overwrites new with old and just replacing names. It doesn't keep any that happened to already have the same name no new or were only on new but not on old. 

Should we consolidate?

Reply:
if we move to what AKS does then we need a differenent way to see if things have changed.

Reply:
Aks and aks-engine label/annotation copy have divereged too much so I gave up on this.